static/src/stylesheets/module/content/_article-immersive.scss

```diff
@@ -112,15 +112,10 @@
     font-weight: 200;
 }
 
-.immersive-main-media__headline-container--dark {
-    .content__headline--immersive-article {
-        color: #ffffff;
-    }
-}
-
```

Comment:
But as far as I can tell, we always want it to be white. So this fix doesn't work, as if something doesn't get `--dark` it breaks.

After doing some digging I'm now unsure if we even need `--dark`, it doesn't override or add anything that we wouldn't want all templates getting.